app/controllers/cart_controller.rb

```diff
@@ -3,7 +3,7 @@ class CartController < ApplicationController
   def show
     item_ids = @cart.contents.keys
     @items = Item.find(item_ids)
-    @total = @cart.contents.map {|item_id, quantity| p Item.find(item_id.to_i).price * quantity}.sum
+    @total = @cart.contents.map {|item_id, quantity| Item.find(item_id.to_i).price * quantity}.sum
```

Comment:
Line is too long. [98/80]<br>Space between { and | missing.<br>Space missing inside }.